script/releases/v1.0.0-slashing/5-executeUpgradeAndUnpause.sol

```diff
@@ -0,0 +1,58 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.12;
+
+import "../Env.sol";
+import {QueueAndUnpause} from "./2-queueUpgradeAndUnpause.s.sol";
+
+import "@openzeppelin/contracts/proxy/transparent/TransparentUpgradeableProxy.sol";
+import "@openzeppelin/contracts/proxy/transparent/ProxyAdmin.sol";
+
+contract Execute is QueueAndUnpause {
+    using Env for *;
+
+    function _runAsMultisig() prank(Env.protocolCouncilMultisig()) internal override {
+        bytes memory calldata_to_executor = _getCalldataToExecutor();
+
+        TimelockController timelock = Env.timelockController();
+        timelock.execute({
+            target: Env.executorMultisig(),
+            value: 0,
+            payload: calldata_to_executor,
+            predecessor: 0,
+            salt: 0
+        });
+    }
+
+    function testScript() public virtual override {
+        runAsEOA();
+
+        TimelockController timelock = Env.timelockController();
+        bytes memory calldata_to_executor = _getCalldataToExecutor();
+        bytes32 txHash = timelock.hashOperation({
+            target: Env.executorMultisig(),
+            value: 0,
+            data: calldata_to_executor,
+            predecessor: 0,
+            salt: 0
+        });
+
+        assertFalse(timelock.isOperationPending(txHash), "Transaction should NOT be queued.");
+
+        // 1- run queueing logic
+        super._runAsMultisig();
+        _unsafeResetHasPranked(); // reset hasPranked so we can use it again
+
+        assertTrue(timelock.isOperationPending(txHash), "Transaction should be queued.");
+        assertFalse(timelock.isOperationReady(txHash), "Transaction should NOT be ready for execution.");
+        assertFalse(timelock.isOperationDone(txHash), "Transaction should NOT be complete.");
+        
+        // 2- warp past delay
+        vm.warp(block.timestamp + timelock.getMinDelay()); // 1 tick after ETA
+        assertEq(timelock.isOperationReady(txHash), true, "Transaction should be executable.");
+        
+        // 3- execute
+        execute();
+
+        assertTrue(timelock.isOperationDone(txHash), "Transaction should be complete.");
```

Comment:
Can we validate ALM has been initialized here?

Reply:
[15671ad](https://github.com/Layr-Labs/eigenlayer-contracts/pull/916/commits/15671adba6dcec1dc4ece525af44794792c775b7)